usagov/vote-gov-drupal
# VOTE-1004: Improve registration tool filtered results

## Jira ticket

[VOTE-1004](https://cm-jira.usa.gov/browse/VOTE-1004)

## Description

Improves registration tool filter functionality so that input can match the state abbreviation or the first letters of words.

## Deployment and testing

### Pre-deploy

1. Insert any required steps to take before merging and running post-deploy.

### Post-deploy

1. Insert any required steps to take before beginning test.

### QA/Test

1. lando retune
2. npm run build
3. Search states using registration tool by typing their state abbreviation
4. Search states by typing their name and confirm results match the first letters of search (https://vote.gov/ state selection tool can be used as reference for expected results)

## Checklist for the Developer




## Checklist for the Peer Reviewers
